protocol/triple/health/health_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package health
+
+import (
+	"sync"
+	"testing"
+	"time"
+)
+import (
+	"github.com/stretchr/testify/assert"
+
+	_ "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+import (
+	healthpb "dubbo.apache.org/dubbo-go/v3/protocol/triple/health/triple_health"
+)
+
+const testService = "testService"
+
+// If there is a conflict between the healthCheck of Dubbo and the healthCheck of gRPC, an error will occur.
+func TestProtoConflict(t *testing.T) {
```

Comment:
This conflict testing is finished by ```_ "google.golang.org/grpc/health/grpc_health_v1"```?
If yes, just leave a comment about this and do not need to introduce ```TestProtoConflict```.